README.md

```diff
@@ -22,8 +22,8 @@ Include a minified browser bundle directly in your HTML like so:
 
```

Comment:
Unrelated to the line:  As of https://github.com/algorand/js-algorand-sdk/pull/728/commits/e8a04c3c4cc6ef3e666d008bee539cf8c327e8d6, the PR shows _This branch is out-of-date with the base branch_.  Is it intentional or should the PR's branch be updated prior to merge?

I'll hold my approval until clarifying intended state.

Reply:
Yes it is intentional.

Reply:
Resolving - thanks for confirmation.